cclib/parser/molcasparser.py

```diff
@@ -40,13 +40,30 @@ def normalisesym(self, label):
     def before_parsing(self):
         # Compile the regex for extracting the element symbol from the
         # atom label in the "Molecular structure info" block.
-        self.re_atomelement = re.compile('([a-zA-Z]+)\d+')
+        self.re_atomelement = re.compile('([a-zA-Z]+)|d+')
 
         # Compile the dashes-and-or-spaces-only regex.
         self.re_dashes_and_spaces = re.compile('^[\s-]+$')
 
+    def _assign_coreelectrons_to_element(self, element, ncore, divide_by_count=False):
```

Comment:
+1 although this looks more like a generic helper function that can live at module level (maybe in a utils module?), since it doesn't use anything in the parser class. Also, please document the intended use and applicability in the docstring.

Reply:
Move to logfileparser.py in #636.